### DISCUSSION TITLE
Restoration loop on empty database

### DISCUSSION THREAD

**Discussion by R0N1n-dev** [supabase/supabase#22314] [General]
Have a project setup on free tier and am using the Auth. It got paused and on trying to restore it, it just doesn't go anywhere. Have tried everyday fro the past 2 weeks and the thing just hangs. 
![Screenshot from 2024-03-28 13-10-29](https://github.com/supabase/supabase/assets/83344872/c67d9cde-8f2c-4265-b7b9-06dbbf703375)

What is this, the database is empty, how can it still be restoring for over half an hour. How can it restore nothing for so long.
Quite frankly, maybe you can come up with an explanation on a database that has thousands of entries. But mine is empty, This is just confusing. I have databases on Mongodb that exceed 5000 entries and that thing restores in a second. Is this  oz its free tier or what????????. If you want me to pay, just say so.!!!!!!

Make it make sense???????? coz its just silly at this point

**Comment by aaronjconway** [comment]
I have  the same. I'm not sure what the point of a free tier is if there is a random possibility of complete data loss and an unusable project after a week. Just don't have a free tier - or trial it.
